catalyst_voices/apps/voices/lib/pages/overall_spaces/space/workspace_overview.dart

```diff
@@ -13,25 +23,105 @@ class WorkspaceOverview extends StatelessWidget {
       child: Column(
         children: [
           SpaceOverviewHeader(Space.workspace),
-          SectionHeader(title: Text('My private proposals (3/5)')),
-          VoicesNavTile(
-            name: 'My first proposal',
-            status: ProposalStatus.draft,
-            trailing: MoreOptionsButton(),
-          ),
-          VoicesNavTile(
-            name: 'My second proposal',
-            status: ProposalStatus.ready,
-            trailing: MoreOptionsButton(),
-          ),
-          VoicesNavTile(
-            name: 'My third proposal',
-            status: ProposalStatus.ready,
-            trailing: MoreOptionsButton(),
-          ),
+          _BrowseMyProposals(),
+          SizedBox(height: 56),
           VoicesDivider(indent: 0, endIndent: 0, height: 16),
+          Expanded(
+            child: SingleChildScrollView(
+              child: _NotPublishedProposalSelector(),
+            ),
+          ),
         ],
       ),
     );
   }
 }
+
+class _BrowseMyProposals extends StatelessWidget {
+  const _BrowseMyProposals();
+
+  @override
+  Widget build(BuildContext context) {
+    return SpaceOverviewNavTile(
+      leading: VoicesAssets.icons.briefcase.buildIcon(),
+      title: Text(
+        context.l10n.browseMyProposals,
+        style: Theme.of(context).textTheme.bodyLarge,
+      ),
+      onTap: () => GoRouter.of(context).go(const MyProposalsRoute().location),
```

Comment:
Would that one work?
```suggestion
      onTap: () => const MyProposalsRoute().go(context),
```